poc_iot_verifier/src/poc_report.rs

```diff
@@ -86,21 +96,49 @@ impl Report {
         .map_err(Error::from)
     }
 
+    pub async fn delete_report<'c, 'q, E>(executor: E, id: &'q Vec<u8>) -> Result
+    where
+        E: sqlx::Executor<'c, Database = sqlx::Postgres> + Clone,
+    {
+        sqlx::query(
+            r#"
+            delete from poc_report
+            where id = $1
+            "#,
+        )
+        .bind(id)
+        .execute(executor.clone())
+        .await
+        .map(|_| ())
+        .map_err(Error::from)
+    }
+
     pub async fn get_next_beacons<'c, E>(executor: E) -> Result<Vec<Self>>
     where
         E: sqlx::Executor<'c, Database = sqlx::Postgres>,
     {
         sqlx::query_as::<_, Self>(
             r#"
-            select * from poc_report
-            where report_type = 'beacon' and status = 'pending'
-            and report_timestamp < (NOW() - INTERVAL '$1 MINUTES')
+            select  poc_report.id,
+                    poc_report.remote_entropy,
```

Comment:
remove the `poc_report.` prefix.. only needed when there's ambiguity in what the source of the field is? Fine post merge

Reply:
fixed here: 24731f41f66861269b324a5fbe5a29ae792f50e8